fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,289 @@
+                      ):
+    """Visualization for metrics with statistical error bounds.
+
+    Plots a given metric and its given error (as described by the `error_bars` or `conf_intervals`)
+
+    This function takes in a :class:`fairlearn.metrics.MetricFrame` with precomputed metrics
+    and metric errors and a `error_bars` or `conf_intervals` array to interpret the columns
+    of the :class:`MetricFrame`.
+
+    The items at each index of the given `metrics` array and given `error_bars` or `conf_intervals`
+    array should correspond to a pair of the same metric and metric error, respectively.
+
+    Note:
+        If given many metrics to plot, this function plots them in only 1 row
+        with the number of subplot columns matching the number of metrics
+
+    Parameters
+    ----------
+    metric_frame : fairlearn.metrics.MetricFrame
+        The collection of disaggregated metric values, along with the metric errors.
+
+    kind : str, optional
+        The type of plot to display. i.e. "bar", "line", etc.
+        List of options is detailed in :meth:`pandas.DataFrame.plot`
+
+    metrics : str or list of str
+        The name of the metrics to plot. Should match columns from the given :class:`MetricFrame`.
+
+    error_bars : str or list of str
+        The name of the error metrics to plot.
+        Should match columns from the given :class:`MetricFrame`.
+        Error bars quantify the bounds relative to the base metric.
+
+        Example:
+            If the error bar for a certain column is :code:`[0.01, 0.02]`
+            and the metric is 0.6
+            Then the plotted bounds will be :code:`[0.59, 0.62]`
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper errors. i.e. :code:`[0.01, 0.02]`
+
+    conf_intervals : str or list of str
+        The name of the confidence intervals to plot.
+        Should match columns from the given :class:`MetricFrame`.
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper bounds. i.e. :code:`[0.59, 0.62]`
+
+    axs : :class:`matplotlib.axes._axes.Axes`\
+        or list of :class:`matplotlib.axes._axes.Axes`, optional
+        Custom Matplotlib axes to which this function can plot
+
+    show_plot : bool, optional
+        Whether to show the plot or not
+
+    plot_error_labels : bool, optional
+        Whether or not to plot numerical labels for the error bounds
+
+    text_precision_digits : int, optional
+        The number of digits of precision to show for error labels
+
+    text_font_size : int, optional
+        The font size to use for error labels
+
+    text_color : str, optional
+        The font color to use for error labels
+
+    text_ha : str, optional
+        The horizontal alignment modifier to use for error labels
+
+    Returns
+    -------
+    :class:`numpy.ndarray` of :class:`matplotlib.axes._axes.Axes`
```

Comment:
These aren't very similar or interchangeable. Can we describe in which case we return which one?

Reply:
Isn't it returning an array of Axes, not either/or?

Actually, I'm not sure that it's always a numpy array of axes - it looks like in some cases it might just be a list. Can you check @alexquach ?

Reply:
List seems intuitive. Why would one use a numpy array to group axes together?

Reply:
I think that's a question for `pandas`:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.html
(look at the 'Returns' section)